### ISSUE TITLE
[Public Catalog] Update the “View Repo” button to point to the correct location for integration packs

### ISSUE THREAD

**Issue by jpvajda** [newrelic/developer-website#1514]
## Description
As the directory structure of the Repo has changed for the **_integration packs_** nested under sub-directories (e.g. `packs/aws/aws-rds`, we will need to account for this when linking back to the repo via the `View Repo` button on the [ObservabilityPackDetails](https://github.com/newrelic/developer-website/blob/feature/o11y-packs/src/templates/ObservabilityPackDetails.js) template page.

Currently we use a [get-pack-url helper function](https://github.com/newrelic/developer-website/blob/feature/o11y-packs/src/utils/get-pack-url.js) to parse the location of the pack in the repo from the `logo.svg`. This is not necessary anymore as there is now a `packUrl` field in NerdGraph that contains this information.

We should move to using `packUrl` for all packs and remove the helper function.

<img width="1413" alt="Screen Shot 2021-08-06 at 4 42 39 PM" src="https://user-images.githubusercontent.com/1395158/128577786-6c31e7cc-1a67-48bf-a9df-88a949e49e65.png">

`Note:` this will require the migration work to be completed as the `packUrl `field is available to reference this pack's location on github.

## Acceptance Criteria 
- [x] All packs should now use the `packUrl` NerdGraph value instead of parsing the url using the helper function
- [x] Ensure the helper function is not required elsewhere and remove

**Comment by nr-kkenney** [comment]
this file contains the helper method that the view button uses: https://github.com/newrelic/developer-website/blob/feature/o11y-packs/src/utils/get-pack-url.js

so updating the implementation there to use packUrl is the way to go for this ticket 👍

**Comment by rudouglas** [comment]
The work here has been completed, the only thing left is checking that the new nested packUrls work as expected

**Comment by jpvajda** [comment]
@rudouglas what's keeping us from testing this and marking this done? Anything in particular?

**Comment by rudouglas** [comment]
Just the new packs being merged in to the main repo and pushed to NerdGraph @jpvajda though we could break that out to separate ticket, or a `nit` to make sure it works? The current implementation works so I see no reason why it also wouldn't work with the nested packs

**Comment by rudouglas** [comment]
I've moved that check to the Nit List as follow-up work so this can be closed once the PR is merged:

https://github.com/newrelic/newrelic-observability-packs/issues/144
